tesla-hero-slider/src/components/slide-params.js

```diff
@@ -0,0 +1,86 @@
+import React, {Component} from 'react';
+import propTypes from 'prop-types';
+import AnimateValue from './animate-value';
+import './../styles/scss/components/slide-params.css';
+
+
+class SlideParams extends Component {
+
+	static propTypes = {
+		activeCar: propTypes.object.isRequired,
+		animationForward: propTypes.bool.isRequired
+	};
+
+	state = {
+		delayTopSpeed: 0,
+		delayMph: 0,
+		delayMileRange: 0
+	};
+
+	componentWillReceiveProps(props) {
+		let delay = 500,
+			delayTopSpeed = delay,
+			delayMph = delay + 250,
+			delayMileRange = delay + 500;
+
+		if (!props.animationForward) {
+			delayTopSpeed = delay + 500;
+			delayMileRange = delay;
+		}
+
+		this.setState({
+			delayTopSpeed,
```

Comment:
You don't need to keep this information in the state. These values are static so just keep it at the top of the module:

```js
const DELAY_TOP_SPEED = 500;
const DELAY_MPH = 750;
const DELAY_MILE_RANG = 1000;
```